### ISSUE TITLE
First-launch wizard starts every time AdGuard is started

### ISSUE THREAD

**Issue by GauthierPLM** [AdguardTeam/AdguardForMac#748]
### Steps to reproduce

1. Install AdGuard 2.5.0.892 nightly (CL-1.7.114) and MacOS 10.15.6 (19G2021) through Brew' cask (did not test other method).
2. Start AdGuard and complete the installation wizard with all default settings.
3. Quit AdGuard or reboot the computer.
4. Start AdGuard (or let it start with the OS): the wizard is run once again. This happens no matter how many time I restart AdGuard.

### Expected behavior
The first-launch wizard should be only run once at first launch. Not every time AdGuard is booting.

### Actual behavior
The first-launch wizard constantly start when starting AdGuard either after a reboot or manually.
Changing the settings in the wizard do not seem to change the behaviour.j

<details><summary>Screenshot:</summary>

Video demonstrating the issue:
https://youtu.be/9nUDTHyEikA

</details>

### Customer ID 
1708806

### Your environment

* AdGuard 2.5.0.892 nightly (CL-1.7.114)
* MacOS 10.15.6 (19G2021)
* TrendMicro Apex One (Mac) Security Agent

**Comment by zzebrum** [comment]
@GauthierPLM 
in the Terminal.app tun these two commands:
```
defaults read com.adguard.mac.adguard "FirstRunV2"
ls -al  ~/Library/Preferences/com.adguard.mac.adguard.plist
```

make a screenshot of the output and post it either here or send to apple[at]adguard[dot]com

**Comment by GauthierPLM** [comment]
Hello,
Here is the requested screenshot:

![Capture d’écran 2020-10-01 à 15 14 06](https://user-images.githubusercontent.com/2579741/94813808-c497fd80-03f8-11eb-9d1d-c74cd23b8354.png)

**Comment by zzebrum** [comment]
@GauthierPLM AdGuard can't save settings to the .plist file due to a lack of permission.
Completely close the AdGuard app, then remove the file: 

`sudo rm /Users/gpogam-lemontagner/Library/Preferences/com.adguard.mac.adguard.plist`

**Comment by GauthierPLM** [comment]
Hello,
It indeed fixed the issue.
I have no idea how the writes of this file changed.
Thank you !

**Comment by GauthierPLM** [comment]
Hello,

The issue happened again after the last update (2.5.0.900 nightly (CL-1.7.119)). The screenshot and results above are the same. I fixed it once again by deleting the file.

Is possible that the installer, which runs as root, changes this file but do not preserve its rights?

**Comment by Chinaski1** [comment]
Hello!

This does not seem to be an internal AdGuard issue.
Try to repeat the manipulations that you have been recommended above.

**Comment by GauthierPLM** [comment]
Hi @Chinaski1

As I said, I did it again and it fixed the issue. However, I am quite surprised it happens: I checked my preference folder and the rights are good (nothing set to root). Also, uninstalling completely AdGuard (`brew cask zap adguard-nightly`) and installing it again do not fix the issue. The fact that the issue happened again just after the update points me to AdGuard' updater as the responsible of this issue…
